spec/type.dd

```diff
@@ -188,6 +191,15 @@ f = Foo.E;       // OK
 -------------------
 )
 
+    $(UL
+    $(LI All types implicitly convert to $(RELATIVE_LINK2 noreturn, `noreturn`).)
+    $(LI Arrays implicitly convert to $(DDSUBLINK spec/arrays, void_arrays, `void[]`).)
```

Comment:
Perhaps specify Static and Dynamic arrays, but not Associative arrays

Reply:
```suggestion
    $(LI Static and dynamic arrays implicitly convert to $(DDSUBLINK spec/arrays, void_arrays, `void[]`).)
```